x-pack/test/security_api_integration/session_idle.config.ts

```diff
@@ -51,6 +51,8 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
             saml_disable: { order: 3, realm: 'saml1', session: { idleTimeout: 0 } },
           },
         })}`,
+        // Exclude Uptime tasks to not interfere (additional ES load) with the session cleanup task.
+        `--xpack.task_manager.unsafe.exclude_task_types=${JSON.stringify(['UPTIME:*'])}`,
```

Comment:
__NOTE:__ This task is [relatively new](https://github.com/elastic/kibana/pull/119359/files), that might explain why we don't have issues with these tests in older versions. I cannot say I like this workaround, but can live with that knowing that we have this only for two test suites.

What do you think?

Reply:
TIL about `exclude_task_types`.

It makes me a bit nervous because the docs state _"This configuration is experimental, unsupported, and can only be used for temporary debugging purposes because it causes Kibana to behave in unexpected ways."_

I'd rather have these tests run than have them skipped, though. And I can't think of a better alternative.

Reply:
> It makes me a bit nervous because the docs state "This configuration is experimental, unsupported, and can only be used for temporary debugging purposes because it causes Kibana to behave in unexpected ways."

Yeah, I share the same feeling. I noticed it's already used in the integration tests, so if we ever decide to drop support for it we'll figure out alternative then (assuming `exclude_task_types` actually helps...): https://github.com/elastic/kibana/blob/0b1425b9741199a3fff4563ee7780ea564698103/x-pack/test/alerting_api_integration/common/config.ts#L174